subsys/mpsl/cx/nrf700x/mpsl_cx_nrf700x.c

```diff
@@ -285,7 +285,7 @@ static int mpsl_cx_init(const struct device *dev)
 #if DT_NODE_HAS_PROP(CX_NODE, req_gpios)
 	uint8_t req_pin = NRF_DT_GPIOS_TO_PSEL(CX_NODE, req_gpios);
 
-	soc_secure_gpio_pin_mcu_select(req_pin, NRF_GPIO_PIN_MCUSEL_NETWORK);
+	soc_secure_gpio_pin_mcu_select(req_pin, NRF_GPIO_PIN_MCU_NETWORK);
```

Comment:
Shouldn't it be `NRF_GPIO_PIN_SEL_NETWORK`? https://github.com/NordicSemiconductor/nrfx/blob/v2.9.0/hal/nrf_gpio.h#L169

Applies to other occurrences here and in `subsys/mpsl/cx/thread/mpsl_cx_thread.c`.

Reply:
nope, as you can see in the description

> @note This enumerator is deprecated. Use @ref nrf_gpio_pin_sel_t instead

Reply:
Exactly. [The source code for `nrf_gpio_pin_sel_t` type](https://github.com/NordicSemiconductor/nrfx/blob/v2.9.0/hal/nrf_gpio.h#L152-L159) :
```c
/** @brief Enumerator used for selecting the MCU/Subsystem to control the specified pin. */
typedef enum
{
    NRF_GPIO_PIN_SEL_APP        = GPIO_PIN_CNF_MCUSEL_AppMCU,     ///< Pin controlled by Application MCU.
    NRF_GPIO_PIN_SEL_NETWORK    = GPIO_PIN_CNF_MCUSEL_NetworkMCU, ///< Pin controlled by Network MCU.
    NRF_GPIO_PIN_SEL_PERIPHERAL = GPIO_PIN_CNF_MCUSEL_Peripheral, ///< Pin controlled by dedicated peripheral.
    NRF_GPIO_PIN_SEL_TND        = GPIO_PIN_CNF_MCUSEL_TND,        ///< Pin controlled by Trace and Debug Subsystem.
} nrf_gpio_pin_sel_t;
```
That's why I suggested to use `NRF_GPIO_PIN_SEL_NETWORK` in this line.

Reply:
Ahh right, I quickly look at this failure and miss that is still deprecated enum used. Thanks for pointing out ;)